Sources/SparkConnect/expressions.pb.swift

```diff
@@ -1887,15 +1887,11 @@ extension Spark_Connect_Expression: SwiftProtobuf.Message, SwiftProtobuf._Messag
     var _common: Spark_Connect_ExpressionCommon? = nil
     var _exprType: Spark_Connect_Expression.OneOf_ExprType?
 
-    #if swift(>=5.10)
```

Comment:
Hmm, wonder why it removed the version check condition.

Reply:
Yes, it's removed because `Swift-GRPC` supports Swift 5.10+ only now, @viirya .
- https://github.com/grpc/grpc-swift?tab=readme-ov-file#versions

Swift 6.2 feature freeze was started too and `Swift-GRPC` will support Swift 6.1+ only when 6.2 is out.
- https://forums.swift.org/t/swift-6-2-release-process/78371

Reply:
BTW, `Apache Spark Connect for Swift` starts with Swift 6.0+.